plugins/community/neoeinstein-prost/v0.2.1/buf.plugin.yaml

```diff
@@ -1,7 +1,7 @@
 version: v1
 name: buf.build/community/neoeinstein-prost
 plugin_version: v0.2.1
-source_url: https://crates.io/crates/protoc-gen-prost
+source_url: https://github.com/neoeinstein/protoc-gen-prost
```

Comment:
There is an argument to be made that the correct source is the plugin installation, and not the GitHub repository.

Reply:
I wonder if in addition to `source_url` we can have a `homepage` that links to the npm registry, go proxy docs, crates.io, etc.